### ISSUE TITLE
Support project search/filtering by tags

### ISSUE THREAD

**Issue by ScottChapman** [DependencyTrack/dependency-track#926]
Tags are nice, but there is no way to search/filter for projects by tags in the UI.

**Comment by stevespringett** [comment]
Yes there is. click a tag of a project and it will show you only the projects with that tag. This works very similar to how GitHub itself works. If you're looking for something different, please be very specific about what you want.

**Comment by ScottChapman** [comment]
ah, I see. that does work! I think what I was looking for was something on the search page, but it's less important if the mechanism exists.
A solution would be to have a `tags` field in search. Then one could simply select the tag there.

**Comment by stevespringett** [comment]
Yes, I think there's room for improvement. I think if you want to turn this enhancement request into a search by tag title, then that's something that would be specific and actionable.

**Comment by ScottChapman** [comment]
Sorry, what I meant above was to have a `tags` column on the project search table that you could make visible. And it would have the list of tags that one could simply click like one would do on the page for a specific project.

**Comment by msymons** [comment]
See also ##238 and 586 (tag management).  This latter covers some of the problems which could cause problems for searching...  ie, dont want to have tags "scott" and "SCOTT" and "Scott", want to control who can create tags,  want to be able to delete tags (tidy up).

**Comment by ckazimie** [comment]
@stevespringett you wrote it is possible to click on a single tag to view only projects with that tag. Is there an url where you can combine 2 tags so that you can view the projects that include both tags?

**Comment by jhrv** [comment]
We are using tags extensively, and would be very nice to have a way of combining multiple tags when searching projects.
